content/docs/command-reference/exp/show.md

```diff
@@ -53,6 +53,20 @@ sorted inside each group, chronologically by default. The `--sort-by` and
 `--sort-order` options can change this ordering, based on any single, visible
 metric or param.
 
+### Parallel Coordinates Plot
+
```

Comment:
```suggestion
```

Reply:
NOTE: I just realized we'll need to remove links to this section though. I think that's best. Replace them for links to [/doc/user-guide/experiment-management/comparing-experiments#parallel-coordinates-plot](https://dvc-org-dvc-parallel-co-flymou.herokuapp.com/doc/user-guide/experiment-management/comparing-experiments#parallel-coordinates-plot) please.